lib/AbstractReaderWriter.js

```diff
@@ -25,22 +25,34 @@ class AbstractReaderWriter extends AbstractReader {
 	 * Writes the content of a resource to a path.
 	 *
 	 * @public
-	 * @param {module:@ui5/fs.Resource} resource The Resource to write
+	 * @param {module:@ui5/fs.Resource} resource Resource to write
+	 * @param {Object} [options]
+	 * @param {boolean} [options.readOnly=false] Whether the resource content shall be written read-only
```

Comment:
I'm not completely sure whether I have understood the use case and behavior of `readOnly`. Where will this be used?